operate/qa/util/src/main/java/io/camunda/operate/qa/util/TestContainerUtil.java

```diff
@@ -300,7 +299,9 @@ public void startElasticsearch(final TestContext testContext) {
                 String.format(
                     "%s:%s",
                     DOCKER_ELASTICSEARCH_IMAGE_NAME,
-                    ElasticsearchClient.class.getPackage().getImplementationVersion()))
```

Comment:
🙃 We can't use the old way, as the new client doesn't has the implementation version in its MANIFEST
![2025-03-05_20-23](https://github.com/user-attachments/assets/3bd966ef-4412-4124-bd67-d2da8ba331cf)